.docker/Dockerfile-build

```diff
@@ -17,28 +18,14 @@ COPY . .
 
 RUN go build -tags sqlite,json1 -o /usr/bin/hydra
 
-FROM alpine:3.18
+#########################
 
-RUN addgroup -S ory; \
-    adduser -S ory -G ory -D  -h /home/ory -s /bin/nologin; \
-    chown -R ory:ory /home/ory
+FROM gcr.io/distroless/static-debian11:nonroot as runner
```

Comment:
Did we check whether this has a compatible glibc, i.e. works with SQLite? Note that here we don't use musl, so the binary has to be linked against glibc.

Reply:
From what I remember you have to set `CGO_ENABLED 0` for Go to work in non-libc environments.